tests/decorators/tests.py

```diff
@@ -8,7 +8,7 @@
 from django.http import HttpRequest, HttpResponse, HttpResponseNotAllowed
 from django.middleware.clickjacking import XFrameOptionsMiddleware
 from django.utils.decorators import method_decorator
-from django.utils.functional import allow_lazy, lazy
```

Comment:
we should probably keep at least once test for `allow_lazy` to ensure we don't break it during the deprecation period.